### PR TITLE
[stable/newrelic-infrastructure] update to 1.6.0, add psp support

### PR DESCRIPTION
**Pull request by verwilst**
#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: stable/newrelic-infrastructure/templates/clusterrolebinding.yaml [hunk 1]
```diff
@@ -1,21 +1,5 @@
 {{- if .Values.rbac.create }}
 apiVersion: rbac.authorization.k8s.io/v1beta1
```
Comment (davidkarlsen) [inline stable/newrelic-infrastructure/templates/clusterrolebinding.yaml]: this can be v1

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

**Comment by verwilst** [comment]
@davidkarlsen Should be ok now :-)

**Comment by verwilst** [comment]
bump

**Review by davidkarlsen — APPROVED** [review]

**Pull request merged**